src/multi-selector.js

```diff
@@ -99,10 +100,16 @@ const MultiSelector = React.createClass({
 		setTimeout(this.triggerItemChange);
 	},
 
+	// Default implementation
 	getItemTitle: function(item) {
 		return item.label;
 	},
 
+	// Default implementation
+	searchValueToItem(searchValue) {
+		return {label: searchValue}
```

Comment:
This is different then the [previous implementation](https://github.com/CanopyTax/cpr-multiselect/pull/126/files#diff-fdefbd18fd65f385533a67ca0bab10d7L245), but I think it makes sense because of the default implementation of getItemTitle.

What are your thoughts?